victor_moveit_config/config/victor.srdf

```diff
@@ -52,7 +52,7 @@
         <group name="right_arm"/>
     </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
-    <group_state name="right_arm_home" group="right_arm">
+    <group_state name="right_arm_zero" group="right_arm">
```

Comment:
Just FYI I've made a lot of named positions for Victor:
https://github.com/UM-ARM-Lab/arm_robots/blob/f1b11b0d7a044ca83b503a694219972f2835e6cb/arm_robots/src/arm_robots/config/victor_config.py#L69

Defining these in python makes the a bit further from Moveit (bad?) but also much more concise to define and edit.

Defining `zero` here seems fine regardless

Reply:
another advntage of defining them in the SRDF, is in the rviz plugin you can select them and plan/go to them. Very useful!

Reply:
I think we should PR the ones you've defined into the SRDF